### ISSUE TITLE
`keepdim` not ignored in torch.argmin when `dim=None`

### ISSUE THREAD

**Issue by lazycal** [pytorch/pytorch#78791]
### 🐛 Describe the bug

In https://pytorch.org/docs/stable/generated/torch.argmin.html?highlight=argmin#torch.argmin:
![image](https://user-images.githubusercontent.com/7333325/171777182-28b50e6c-1a0c-48b9-a389-cc1d1591f8b0.png)

But passing `keepdim` with different values returns different results, as shown by the following script:
```python
import torch

a = torch.rand(10, 3)
print(torch.argmin(a, dim=None, keepdim=False))  # output: tensor(27)
print(torch.argmin(a, dim=None, keepdim=True))  # output: tensor([[27]])
```
where the ranks differ. It seems like a bug as I feel the second one should also return `tensor(27)`.

### Versions

```
Collecting environment information...
PyTorch version: 1.13.0a0+gitb651148
Is debug build: False
CUDA used to build PyTorch: Could not collect
ROCM used to build PyTorch: N/A

OS: Ubuntu 18.04.5 LTS (x86_64)
GCC version: (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
Clang version: 11.1.0-++20211011094159+1fdec59bffc1-1~exp1~20211011214614.8
CMake version: version 3.19.6
Libc version: glibc-2.17

Python version: 3.7.11 (default, Jul 27 2021, 14:32:16)  [GCC 7.5.0] (64-bit runtime)
Python platform: Linux-5.4.0-90-generic-x86_64-with-debian-buster-sid
Is CUDA available: False
CUDA runtime version: 11.2.152
GPU models and configuration: GPU 0: NVIDIA GeForce RTX 2080
Nvidia driver version: 495.29.05
cuDNN version: Probably one of the following:
/usr/lib/x86_64-linux-gnu/libcudnn.so.8.1.1
/usr/lib/x86_64-linux-gnu/libcudnn_adv_infer.so.8.1.1
/usr/lib/x86_64-linux-gnu/libcudnn_adv_train.so.8.1.1
/usr/lib/x86_64-linux-gnu/libcudnn_cnn_infer.so.8.1.1
/usr/lib/x86_64-linux-gnu/libcudnn_cnn_train.so.8.1.1
/usr/lib/x86_64-linux-gnu/libcudnn_ops_infer.so.8.1.1
/usr/lib/x86_64-linux-gnu/libcudnn_ops_train.so.8.1.1
HIP runtime version: N/A
MIOpen runtime version: N/A
Is XNNPACK available: False

Versions of relevant libraries:
[pip3] mypy==0.812
[pip3] mypy-extensions==0.4.3
[pip3] numpy==1.21.5
[pip3] onnx-pytorch==0.1.4
[pip3] torch==1.13.0a0+gitb651148
[pip3] torchvision==0.12.0
[conda] blas                      1.0                         mkl  
[conda] cudatoolkit               11.3.1               h2bc3f7f_2  
[conda] magma-cuda110             2.5.2                         1    pytorch
[conda] mkl                       2021.4.0           h06a4308_640  
[conda] mkl-include               2021.4.0           h06a4308_640  
[conda] mkl-service               2.4.0            py37h7f8727e_0  
[conda] mkl_fft                   1.3.1            py37hd3c417c_0  
[conda] mkl_random                1.2.2            py37h51133e4_0  
[conda] numpy                     1.21.5                   pypi_0    pypi
[conda] numpy-base                1.21.2           py37h79a1101_0  
[conda] onnx-pytorch              0.1.4                    pypi_0    pypi
[conda] torch                     1.13.0a0+gitb651148           dev_0    <develop>

```

cc @svekars @holly1238

**Comment by gchanan** [comment]
We should be following numpy semantics, which the code is doing but the documentation isn't:

```
>>> a=np.random.rand(10, 3)
>>> np.argmin(a, axis=None, keepdims=False)
14
>>> np.argmin(a, axis=None, keepdims=True)
array([[14]])
```

**Comment by soulitzer** [comment]
Thanks for the report, we would accept a PR to correct this.
